fury/tests/test_convert.py

```diff
@@ -0,0 +1,35 @@
+import os
+import pytest
+import numpy.testing as npt
+from tempfile import TemporaryDirectory
+from fury.io import load_image
+# Optional packages
+from fury.optpkg import optional_package
+matplotlib, have_matplotlib, _ = optional_package('matplotlib')
+
+if have_matplotlib:
+    import matplotlib.pyplot as plt
+    from fury.convert import matplotlib_figure_to_numpy
+
+
+@pytest.mark.skipif(not have_matplotlib, reason="Requires MatplotLib")
+def test_convert():
+    names = ['group_a', 'group_b', 'group_c']
+    values = [1, 10, 100]
+
+    fig = plt.figure(figsize=(9, 3))
+    # plt.subplot(131)
+    # plt.bar(names, values)
+    # plt.subplot(132)
+    # plt.scatter(names, values)
+    # plt.subplot(133)
+    # plt.plot(names, values)
+    # plt.suptitle('Categorical Plotting')
```

Comment:
why is it commented? your figure seems empty

Reply:
The test failed for some reason when I added these args,  

I think I could just test an empty figure, and confirm it works.
will that be the case? 

I'm not sure...

I don't understand enough about matplotlib, so I can't reliably say why it fails when I uncomment it... 
I'll keep researching & hopefully find a fix.

Any suggestions? any idea why this might be failing? any guess is appreciated...

Reply:
See below the test updated. I admit that it was tricky., you had to update the `matplotlib_figure_to_numpy` option and the `fig.savefig` options to make sure that both images have the same size and orientation.

```python
    fig = plt.figure(figsize=(9, 3))
    plt.subplot(131)
    plt.bar(names, values)
    plt.subplot(132)
    plt.scatter(names, values)
    plt.subplot(133)
    plt.plot(names, values)
    plt.suptitle('Categorical Plotting')
    arr2 = matplotlib_figure_to_numpy(fig, transparent=False,
                                      flip_up_down=False)

    with TemporaryDirectory() as tmpdir:
        fname = os.path.join(tmpdir, 'tmp.png')
        fig.savefig(fname, transparent=False, bbox_inches='tight',
                    pad_inches=0)
        arr1 = load_image(fname)
        npt.assert_array_equal(arr1, arr2)
```